api/src/routes/ical.ts

```diff
@@ -53,11 +55,13 @@ export function icalRoute(pool: Pool): Handler {
 
     for (const ctf of ctfs) {
       cal.createEvent({
+        id: `${ctf.id}@${config.pad.domain}`,
```

Comment:
`config.pad.domain` may be an empty string. Is there a reason you included the domain in the event ID?

Reply:
From the spec:
> The "UID" itself MUST be a globally unique identifier.
      The generator of the identifier MUST guarantee that the identifier
      is unique.  There are several algorithms that can be used to
      accomplish this.  A good method to assure uniqueness is to put the
      domain name or a domain literal IP address of the host on which
      the identifier was created on the right-hand side of an "@", and
      on the left-hand side, put a combination of the current calendar
      date and time of day (i.e., formatted in as a DATE-TIME value)
      along with some other currently unique (perhaps sequential)
      identifier available on the system (for example, a process id
      number).  Using a DATE-TIME value on the left-hand side and a
      domain name or domain literal on the right-hand side makes it
      possible to guarantee uniqueness since no two hosts should be
      using the same domain name or IP address at the same time.  Though
      other algorithms will work, it is RECOMMENDED that the right-hand
      side contain some domain identifier (either of the host itself or
      otherwise) such that the generator of the message identifier can
      guarantee the uniqueness of the left-hand side within the scope of
      that domain.
So as a fallback for an empty domain, we may choose `ctfnote` or something like that.
For further uniqueness we could add the eventid of the ctftimeurl (or even the full url), so that even two instances with the same domain likely do not clash as they would have different internal database ids for the same ctf.

Reply:
I think adding the `ctfnote` fallback is a good idea as well as adding the eventid of the ctftimeurl! It is the case for me that I have multiple CTFNote instances in my ical sync and even though it is for me the case that the domains are correctly configured, it may be better to make the identifier also more unique.

Reply:
Done.